src/providers/EtherspotContextProvider.tsx

```diff
@@ -364,6 +364,41 @@ const EtherspotContextProvider = ({
     [sdk, accountAddress]
   );
 
+  const getRatesByTokenAddresses = useCallback(
+    async (chainId: number, tokenAddresses: string[]) => {
+      let tokens = tokenAddresses.filter((address) => address !== null);
```

Comment:
```suggestion
      const tokens = tokenAddresses.filter((address) => address !== null);
```

Reply:
also consider adding zero address checks, not just null

Reply:
I see this variable called `zeroAddressCinstants` it has a list of addresses

Reply:
i think we can use the function `isZeroAddress`